hivemq-edge/src/frontend/src/modules/Workspace/components/edges/ObservabilityEdgeCTA.spec.cy.tsx

```diff
@@ -0,0 +1,24 @@
+import ObservabilityEdgeCTA from './ObservabilityEdgeCTA.tsx'
+
+describe('ObservabilityEdgeCTA', () => {
+  beforeEach(() => {
+    cy.viewport(400, 400)
+  })
+
+  it.only('should render the button', () => {
+    cy.mountWithProviders(<ObservabilityEdgeCTA source="1" onClick={cy.stub().as('onClick')} />)
+    cy.get('button').eq(0).should('not.have.attr', 'aria-describedby')
+    cy.get('button').eq(0).focus()
+    cy.get('button').eq(0).should('have.attr', 'aria-describedby')
+    cy.getByTestId(`icon-button-tooltip`).eq(0).should('contain.text', 'Open the Observability panel')
+    cy.get('button').eq(0).click()
+    cy.get('@onClick').should('have.been.called')
```

Comment:
[nit]: I see the `click` event on line 14, but you might want to test when the `@onClick` `.not.to.have.been.called` was true, so the test itself can tell a story, and easily catch possible future bugs.

Reply:
You mean making sure it's not.to.have.been.called BEFORE the click? 
Do you think it's a good pattern or an overkill? 
It should be done for all the click and have.been.called

Reply:
Thank you for the question.
It's neither an overkill, nor does it have anything to do with patterns.
It is in fact, the only way to prove that your action on line 14 is what led to the line 15 being true, else, one can argue that if you tested `have.been.called` on line 12, it might have too. It makes it easier to catch nasty bugs.